src/device/router_device_routing.erl

```diff
@@ -80,6 +96,13 @@
 -define(OUI_UNKNOWN, oui_unknown).
 -define(DEVADDR_NO_DEVICE, devaddr_no_device).
 
+%% Packet hash cache
+-define(PHASH_TO_DEVICE_CACHE, phash_to_device_cache).
+-define(PHASH_TO_DEVICE_CACHE_MAXSIZE, 4 * 1024 * 1024).
+-define(PHASH_TO_DEVICE_CACHE_MINSIZE, round(0.3 * ?PHASH_TO_DEVICE_CACHE_MAXSIZE)).
+%% e2qc lifetime is in seconds
+-define(PHASH_TO_DEVICE_CACHE_LIFETIME, 1).
```

Comment:
Shout it be a little longer for late packets just in case?

Reply:
Updated to be 1 second over the rx2_window